modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt

```diff
@@ -44,35 +39,36 @@ class HelpFragment : BaseFragment(), HasBackstack {
         savedInstanceState: Bundle?,
     ) = content {
         AppThemeWithBackground(theme.activeTheme) {
-            val bottomPadding by settings.bottomInset.collectAsState(0)
+            val density = LocalDensity.current
+            val imeInsets = WindowInsets.ime
+            val keyboardHeight = with(density) { imeInsets.getBottom(density).toDp() }
+            
+            val basePadding = 16.dp
+            val keyboardPadding = if (keyboardHeight > 0.dp) {
+                24.dp
+            } else {
+                0.dp
+            }
+            val totalPadding = basePadding + keyboardPadding
```

Comment:
We need to account for previously used `settings.bottomInset`. Otherwise content will be obstructed by the mini player when it is available.

<img width="745" alt="Screenshot 2025-06-11 at 15 29 26" src="https://github.com/user-attachments/assets/96ff99d6-ff65-43ef-bfe1-8af86efa51b8" />